doc/rest-api.md

```diff
@@ -926,7 +926,9 @@ Input (run bash):
         "command": ["/bin/bash"],       # Command and arguments
         "environment": {},              # Optional extra environment variables to set
         "wait-for-websocket": false,    # Whether to wait for a connection before starting the process
-        "interactive": true             # Whether to allocate a pts device instead of PIPEs
+        "interactive": true,            # Whether to allocate a pts device instead of PIPEs
+        "width": 80,                    # initial width of the terminal (optional)
+        "height": 25,                   # initial height of the terminal (optional)
```

Comment:
Complete nitpick I realize but could you replace "initial" by "Initial" (capitalize the first letter), would be more in line with the other ones.